### PR TITLE
chore: separate integrations tab

### PR DESCRIPTION
**Pull request by rpmccarter**

### CODE REVIEW

File: mint.json [hunk 1]
```diff
@@ -40,6 +40,14 @@
       "name": "Components",
       "url": "content"
     },
+    {
+      "name": "Advanced",
+      "url": "advanced"
+    },
```
Comment (handotdev) [inline mint.json]: Can we move advanced to the end? A fair number of folks use the integrations

File: mint.json [hunk 2]
```diff
@@ -40,6 +40,14 @@
...
+    {
+      "name": "Mintlify API",
+      "url": "api-reference"
+    },
```
Comment (handotdev) [inline mint.json]: A bit weird imo to bundle chat into api reference and also with the API reference not being super robust. Can we move these into advanced?

Comment (rpmccarter) [reply mint.json]: Chatted with @hahnbeelee about this - the widget is technically an "API". I initially gave widget its own tab but that looked weird with only one page. Happy to just place this in advanced for now until we have more widget/api features, but then I do think we should separate these two out and give them a dedicated tab

File: mint.json (hunk 1 above)
Comment (rpmccarter) [reply mint.json]: done

**Review by handotdev — APPROVED** [review]
LGTM!

**Pull request merged**